### ISSUE TITLE
Character cannot roll anything

### ISSUE THREAD

**Issue by Stefouch** [pwatson100/alienrpg#17]
I have a strange effect: one of my character cannot roll anything.

Here is the error:
```
Uncaught (in promise) TypeError: Cannot convert undefined or null to object
    at Function.entries (<anonymous>)
    at actor.js:38
    at Array.reduce (<anonymous>)
    at alienrpgActor.getRollData (actor.js:34)
    at alienrpgActor.rollAbility (actor.js:285)
    at alienrpgActorSheet._onRoll (actor-sheet.js:365)
    at HTMLHeadingElement.dispatch (jquery.min.js:2)
    at HTMLHeadingElement.v.handle (jquery.min.js:2)
```

With the debugger: `itemData.attributes` on line 38 is undefined.

line 35: key = `"mobility"`
line 36: itemData = `{ skill: "", description: "<h2>No Stunts Entered</h2><p>Edit the Skill-Stunts.......blablabla....</p>"}`

Any skill roll (and initiative) will do nothing except return this above error (with the same itemData).

Other characters have no problems.

Any clue what could cause this issue ?

**Comment by pwatson100** [comment]
Hi,  I think this was something I found out the other day.  I suspect the player dragged one of the Skill-Stunts on to their character sheet.  They were never really intended for that purpose but just as place holders so the GM could populated them with the Stunts from the book. Then when the player clicks on the spanners next to the stunts it displays the text from the corresponding Skill-Stunt. As an item they have no other attributes hence the error.  I have not had a chance to look at it any further.    
In the update I did the other day I advised GM's to set the permissions of the Skill-Stunts folder to "None" for all players.  That way they cant see them and drag them on to the sheet.   
Trying to correct this is on the todo list.
